README.md

```diff
@@ -1,13 +1,13 @@
 <img src="images/kw_logo.png" width="600" alt="kworkflow">
 
-[![Build Status](https://www.travis-ci.com/kworkflow/kworkflow.svg?branch=unstable)](https://www.travis-ci.com/kworkflow/kworkflow)
+![Build Status](https://github.com/kworkflow/kworkflow/actions/workflows/unit_tests.yml/badge.svg?branch=unstable)
```

Comment:
I'm confused about this part. Is this the official way to get a badge from Github?

Reply:
Yes, I changed it because we stopped using travis. We can add a badge for any workflows we have, I thought the unit tests one would be most appropriate. The second link is just a redirect to be used on click, so I removed it.

More info: https://docs.github.com/en/actions/managing-workflow-runs/adding-a-workflow-status-badge